.github/workflows/codecov.yml

```diff
@@ -28,10 +28,10 @@ jobs:
         run: |
           python setup.py install
```

Comment:
Perhaps, need changes on this part

Reply:
Great point, I've fixed up the CI parts to work with the new changes

NOTE: there are some limitation with python 3.6 that required work arounds